chronicle/src/main/java/net/openhft/chronicle/tcp/SourceTcp.java

```diff
@@ -177,10 +186,10 @@ public void run() {
                 socketChannel.socket().setSoTimeout(0);
                 socketChannel.socket().setSoLinger(false, 0);
 
-                if(builder.receiveBufferSize() > 0) {
+                if (builder.receiveBufferSize() > 0) {
                     socketChannel.socket().setReceiveBufferSize(builder.receiveBufferSize());
                 }
-                if(builder.sendBufferSize() > 0) {
+                if (builder.sendBufferSize() > 0) {
                     socketChannel.socket().setSendBufferSize(builder.sendBufferSize());
                 }
```

Comment:
SocketChannel is configured in both SourceTcp/SinkTcp and TcpEventHandler (i.e. send buffer size, etc) , would be better to have a single or at least not a duplicated place where the socket is configured.